### ISSUE TITLE
ParquetConverter to append to ParquetWriter in chunks

### ISSUE THREAD

**Issue by azurecoder** [elastacloud/parquet-dotnet#372]
**Version:** Parquet.Net v3.2.4

**Runtime Version:** .Net Framework v4.6

**OS:** Windows

#### Expected behavior

Should allow append of CLR types to ParquetWriter if you have a large collection of objects that you want to write to a parquet stream

#### Actual behavior

Calling multiple times will just overwrite the footer 

### Steps to reproduce the behavior

1. Invoke ParquetConverter.Serialize with x rows
2. Invoke ParquetConverter.Serialize with x rows again
3. Read in using Spark or something else, example response:

 java.io.IOException:Expected 2796 values in column chunk at test.log.parquet offset 4 but got 5000 values instead over 1 pages ending at file offset 7222 total entry:9

#### Code snippet reproducing the behavior

```csharp
(payloads.Count % 5000 == 0)
                     {
                        ParquetConvert.Serialize(payloads.ToArray(), fileStream, null, CompressionMethod.Snappy, 5000);
                        _log.Trace($"Chunk of {counter} written to {fileName}");
                        payloads.Clear();
                     }
                  }
 
                  if (payloads.Count > 0)
                  {
                     ParquetConvert.Serialize(payloads.ToArray(), fileStream, null, CompressionMethod.Snappy, payloads.Count);
                     _log.Trace($"Chunk of {counter} written to {fileName}");
                     payloads.Clear();
                  }
```

**Comment by aloneguid** [comment]
released in 3.2.4

**Comment by aloneguid** [comment]
implemented long time ago in 3.2.3, closing
